app/controllers/user_feedbacks_controller.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
```

Comment:
Do we want these to be visible to the site at large? I can see the benefits of having it moderator / admin only, as well as having it publicly available.

Reply:
at some point we'll figure out an interface for viewing feedbacks, but that's a later task. for now we'll just shove them in the database and worry about them later.